{
    "type": "bundle",
    "id": "bundle--b5c8f909-05c5-4f8d-9f25-cba746c61c98",
    "objects": [
        {
            "type": "vulnerability",
            "spec_version": "2.1",
            "id": "vulnerability--74f33f01-538e-4ec6-90da-405e505245f4",
            "created_by_ref": "identity--8ce3f695-d5a4-4dc8-9e93-a65af453a31a",
            "created": "2025-05-13T00:22:34.979347Z",
            "modified": "2025-05-13T00:22:34.979347Z",
            "name": "CVE-2025-31195",
            "description": "The issue was addressed by adding additional logic. This issue is fixed in macOS Sequoia 15.4. An app may be able to break out of its sandbox.",
            "external_references": [
                {
                    "source_name": "cve",
                    "external_id": "CVE-2025-31195"
                }
            ]
        }
    ]
}